src/unix/linux_like/android/b64/x86_64/mod.rs

```diff
@@ -41,6 +42,46 @@ s! {
         pub st_ctime_nsec: ::c_long,
         __unused: [::c_long; 3],
     }
+
+    pub struct _libc_fpxreg {
+        pub significand: [u16; 4],
+        pub exponent: u16,
+        __padding: [u16; 3], // TODO: ignore padding
```

Comment:
What is exactly what remains to be done here ?

The padding will participate in the Debug, Hash, etc. impls, so this type needs to be defined in `s_no_extra_traits!`.